config/environments/development.rb

```diff
@@ -9,14 +9,14 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
-  # config.paperclip_defaults = {
-  #     :storage => :s3,
-  #     :s3_credentials => {
-  #         :bucket => ENV['S3_BUCKET_NAME'],
-  #         :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
-  #         :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
-  #     }
-  # }
+config.paperclip_defaults = {
+     :storage => :s3,
+     :s3_credentials => {
+         :bucket => ENV['AWS_BUCKET'],
+         :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
+         :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
+     }
+ }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.